torch/_inductor/ir.py

```diff
@@ -3111,13 +3111,16 @@ def create(
 
             # CUDA channels_last path depend on cudnn version, see
             # https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ConvUtils.h.
-            valid_device = True
+            valid_cudnn = False
             if (
-                x.get_device() == "cuda"
-                and torch.backends.cudnn.is_available()
-                and torch.backends.cudnn.version() < 8302
+                torch.backends.cudnn.is_available()
+                and torch.backends.cudnn.version() > 8302
```

Comment:
Oh it should be `>= 7603`?

![image](https://user-images.githubusercontent.com/13822661/196589389-f20d962d-fecd-4e15-ae66-2408d02388ec.png)

Reply:
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ConvUtils.h#L324-L330

Reply:
fixed